WebService/v1/Controllers/StatusController.cs

```diff
@@ -219,5 +247,33 @@ private string GetIoTHubMetricsUrl(bool isRunning)
                    $"/resourceGroups/{this.deploymentConfig.AzureResourceGroup}" +
                    $"/providers/Microsoft.Devices/IotHubs/{this.deploymentConfig.AzureIothubName}/Metrics";
         }
+
+        private int GetActiveDeviceCount(bool isRunning)
```

Comment:
Instead of passing isRunning to all functions, can we add isRunning as a private prop and use it in all places ?